drf_braces/fields/modified.py

```diff
@@ -24,6 +24,24 @@ def quantize(self, value):
         return super(DecimalField, self).quantize(value)
 
 
+class CurrencyField(DecimalField):
```

Comment:
better place is probably `custom.py` since `CurrentlyField` does not exist in vanilla DRF